src/main/java/frc/team5115/Commands/Auto/AutoAimAndRangeCommand.java

```diff
@@ -0,0 +1,37 @@
+package frc.team5115.Commands.Auto;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team5115.Classes.Software.AutoAimAndRange;
+
+
+public class AutoAimAndRangeCommand extends Command{
+    private final AutoAimAndRange autoAimAndRange;
+
+
+    double[] errors = {100000000000.0, 10000000000000.0};
+    boolean isBlue;
+    
+    public AutoAimAndRangeCommand(AutoAimAndRange autoAimAndRange) {
+        this.autoAimAndRange = autoAimAndRange;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute(){
+        errors = autoAimAndRange.periodic1();
```

Comment:
Is there a plan to do anything with the errors this returns?

Reply:
I think this was just so the errors are big enough to stop the auto or something like that